### PR TITLE
Des

### PR DESCRIPTION
**Pull request by DesireeWilson**
Okie dokie! I think I figured out how to add reviewers to my pull requests. Also, I believe I fixed the following issues:

- removed path("/") from @Mapping annotations
- refractored @PutMapping updateUser() method
- updated the @DeleteMapping deleteUser() method
- removed findById() method

Hopefully the code is more correct.

### CODE REVIEW

File: demo/src/main/java/com/example/demo/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path="api/v1/user")
+public class UserController {
+
+    @Autowired
+    public UserService userService;
+
+    @PostMapping//create
```
Comment (RuslanZinovyev) [inline demo/src/main/java/com/example/demo/controller/UserController.java]: please remove unnecessary comment // create
it should be obvious from the context, do the same for other methods as well

File: demo/src/main/java/com/example/demo/model/User.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    private Long id;
+    @Column
+    private String firstName;
+    @Column
+    private String lastName;
+    @Column
+    private int age;
+
+    public User() {}
+    public User(String firstName, String lastName, int age) {
```
Comment (RuslanZinovyev) [inline demo/src/main/java/com/example/demo/model/User.java:17]: I recommend you remove super() constructor and follow the same argument sequence as you stated in the User object (id, firstName, lastName)

Comment (RuslanZinovyev) [reply demo/src/main/java/com/example/demo/model/User.java:17]: change getters in setters accordingly in the proper order

File: demo/src/main/java/com/example/demo/services/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public UserRepository userRepository;
+
+    @Override
+    public String createUser(User user) {
+        userRepository.save(user);
+        return "User has been created";
+    }
+
+    @Override
+    public String deleteUser(User user) {
```
Comment (RuslanZinovyev) [inline demo/src/main/java/com/example/demo/services/UserServiceImpl.java]: usually delete method returns boolean, check the return value of delete method in repository

**Review by RuslanZinovyev — APPROVED** [review]

**Comment by DesireeWilson** [comment]
I addressed the comments and now will ask Mr. Ruslan to double check my work. :)

**Pull request closed without merging**